clusterinfo/v1beta1/types.go

```diff
@@ -218,6 +218,11 @@ type OCPDistributionInfo struct {
 	// Controller will sync this field to managedcluster's ManagedClusterClientConfigs
 	// +optional
 	ManagedClusterClientConfig ClientConfig `json:"managedClusterClientConfig,omitempty"`
+
+	// apiServerURL is a valid URI with scheme 'https', address and
+	// optionally a port (defaulting to 443).  apiServerURL can be used by components like the web console
```

Comment:
this is the apiserverurl that has been applied to the managedcluster resource successfully